launcher/errors.go

```diff
@@ -0,0 +1,28 @@
+package launcher
+
+// RetryableError means launcher should reboot the VM to retry.
+type RetryableError struct {
+	Err error
+}
+
+// NonRetryableError means launcher shouldn't reboot the VM to retry.
+type NonRetryableError struct {
+	Err error
+}
+
+// WorkloadError represents the result of an workload/task that is non-zero.
+type WorkloadError struct {
+	ReturnCode uint32
+}
+
+func (e *RetryableError) Error() string {
+	return e.Err.Error()
```

Comment:
maybe we should prefix in case we want to log these: `failed with retryable error:`. same below

Reply:
done